src/vstest.console/CommandLine/Executor.cs

```diff
@@ -222,7 +222,14 @@ private int GetArgumentProcessors(string[] args, out List<IArgumentProcessor> pr
         // Examples: processors to enable loggers that are statically configured, and to start logging,
         // should always be executed.
         var processorsToAlwaysExecute = processorFactory.GetArgumentProcessorsToAlwaysExecute();
-        processors.AddRange(processorsToAlwaysExecute);
+        foreach (var processor in processorsToAlwaysExecute)
+        {
+            if (processors.Any(i => i.Metadata.Value.CommandName == processor.Metadata.Value.CommandName)) {
+                continue;
+            }
```

Comment:
Why is this needed? We grab **processors to always execute** and exclude some of them from execution.